source/main/gui/panels/GUI_TopMenubar.h

```diff
@@ -53,6 +54,23 @@ class TopMenubar
 
     bool IsVisible() { return m_open_menu != TopMenu::TOPMENU_NONE; };
 
+    // Vehicle AI
```

Comment:
That's a lot of AI-only stuff in TopMenubar - it needs it's own GUI window object. Use this commit https://github.com/RigsOfRods/rigs-of-rods/pull/2939/commits/64bb466655eefae46f6ee54b289e5917ecc51e96 as a tutorial for adding new UI object. Visually, keep it as-is in the top menubar.